packages/config/next-i18next.config.js

```diff
@@ -1,42 +1,10 @@
-const path = require("path");
+const i18n = require("../../i18n.json");
 
 /** @type {import("next-i18next").UserConfig} */
 const config = {
   i18n: {
-    defaultLocale: "en",
-    locales: [
-      "ar",
-      "az",
-      "ca",
-      "cs",
-      "da",
-      "de",
-      "en",
-      "es-419",
-      "es",
-      "et",
-      "fr",
-      "he",
-      "hu",
-      "it",
-      "ja",
-      "km",
-      "ko",
-      "nl",
-      "no",
-      "pl",
-      "pt-BR",
-      "pt",
-      "ro",
-      "ru",
-      "sr",
-      "sv",
-      "tr",
-      "uk",
-      "vi",
-      "zh-CN",
-      "zh-TW",
-    ],
+    defaultLocale: i18n.locale.source,
+    locales: i18n.locale.targets.concat([i18n.locale.source]),
```

Comment:
Does the order matter? (The previous one has "en" in the middle of the array)

Reply:
no the order doesn't matter!